features/plugin-list.feature

```diff
@@ -0,0 +1,56 @@
+Feature: List WordPress plugins
+
+  Scenario: Refresh update_plugins transient when listing plugins with --force-check flag
```

Comment:
@baizmandesign What do you think of this Behat test refactor?

Reply:
I ran the behat tests, and they worked for me! Very exciting. A couple comments:

+ I'm unclear why the `wp plugin list --fields=name,status,update` command needs to be executed twice after `update-transient.php` is run.
+ Is it an issue that the version of the Hello Dolly plugin, 1.7.2, is hard-coded in the test?